tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -159,7 +159,6 @@ stages:
         --skip_submodule_sync \
         --parallel \
         --use_nnapi \
-        --build_shared_lib \
```

Comment:
Why do we need to remove this line?

Reply:
Since we test them right after build, we don't need it.